include/graaflib/algorithm/graph_isomorphism.h

```diff
@@ -0,0 +1,482 @@
+#pragma once
+
+#include <algorithm>
+#include <graaflib/types.h>
+#include <graaflib/graph.h>
+#include <vector>
+#include <unordered_map>
+#include <optional>
+#include <unordered_set>
+#include <array>
+
+namespace graaf::algorithm {
+
+// Type aliases
```

Comment:
Following naming conventions, you'd start your alias names with a lowercase letter. You could also make `Mapping` a bit more descriptive, e.g. `vertex_mapping`.